recipes/wavelet_buffer/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake, cmake_layout
+from conan.tools.files import get, copy, rm, rmdir
```

Comment:
Linter

```suggestion
from conan.tools.files import get, copy, rmdir
```

Reply:
Done

Reply:
Sure.

But I believe that all mandatory requirements for PR approval should be clearly described in docs and code should be verified by Ci if it's not too hard. 

See how it looks for the new user:
I bring new recipe which is already workable for different platforms 
I read tutor about how to contribute new packages.
I installed conan hooks and released that recipes must be compatibility with conan 2.0 (which still in beta). 
Fixed code according hooks and CI output
Fixed mostly of highlighted problems except few of them because solution is still not clear for me.
And I'm still waiting third reviewer)

Current documentation is a mix from 1.x and 2.0 approaches which is really not clear.
If you moderate not only what recipes must do but how and even moderate  syntax/layout you should start from CI check. 
Clear documentation -> stable CI -> PR strict requirements , not in opposite direction

Review/fix iterations by at least 3 reviewers in strict but not fully CI supported mode is a pretty resource intensive as for reviewers as for contributors.
This make contributing hard to production due unknown time and work expenses.
Please compare such contribution with PyPI



I'm ready to participate in conan development,  not only to write comments.